processing/src/test/java/org/apache/druid/query/aggregation/last/FloatLastVectorAggregatorTest.java

```diff
@@ -65,14 +65,15 @@ public void setup()
   @Test
   public void initValueShouldBeNegInf()
   {
-    target.initValue(buf, 0);
+    target.init(buf, 0);
```

Comment:
This test looks incorrect. When we call init, the timestamp should be inserted at `position`

```suggestion
    target.initValue(buf, 0);
```